### ISSUE TITLE
Using ANDI with Power Apps

### ISSUE THREAD

**Issue by AndMcI** [SSAgov/ANDI#131]
I am unable to use ANDI for Section 508 testing on Power Apps pages/flyouts. When I try to activate ANDI, it just get the progress spinner and never get back to the page for testing. Is there a way around this or is there a way for ANDI to work on Power Apps pages/flyouts?

![Power Apps page](https://user-images.githubusercontent.com/102681398/160872759-3adbb7c6-6a03-46c6-bec4-14b8bd603436.png)
![Power Apps with ANDI](https://user-images.githubusercontent.com/102681398/160872765-2c1e0bd3-4d17-4951-b3bc-21ab02205a6d.png)

**Comment by JohnCotterSSA** [comment]
Sometimes, depending on how the application's iframes are developed, this issue can occur with ANDI. The SSA 508 test team found a good solution for using ANDI to test iframes when issue like this occur. Give this a try:

- Navigate to the Application Test page (Don't launch ANDI yet)
- Open Dev tools (F12)
- Find and select the <iframe> element in the HTML DOM under the Dev Tools elements tab
- Switch to the Console tab
- Paste the ANDI favelet URL and hit enter.
`javascript:void((function(){andiScript=document.createElement('script');andiScript.setAttribute('src','https://www.ssa.gov/accessibility/andi/andi.js');document.body.appendChild(andiScript)})());`
- This should cause ANDI to be injected into the iframe, and should not trigger reloading of the iframe (which is what is causing this issue).

**Comment by AndMcI** [comment]
Thank you for your response. I am unable to get these steps to work in order to turn ANDI on and test an iFrame.


***@***.***
Andrea McIlvain
Section 508 Specialist
Contractor: Synergy Business & Information Solutions
Digital Infrastructure Services Center
Enterprise Applications Services
Office of the Chief Information Officer
United States Department of Agriculture
Fort Collins, CO
***@***.***


Confidentiality Notice: This electronic message contains information generated by the USDA solely for the intended recipients.  Any unauthorized interception of this message or the use or disclosure of the information it contains may violate the law and subject the violator to civil or criminal penalties.  If you believe you have received this message in error, please notify the sender and delete the email immediately.
